src/reactapp/package.json

```diff
@@ -73,6 +62,8 @@
     "@babel/eslint-parser": "^7.15.0",
     "@babel/preset-env": "^7.15.0",
     "@babel/preset-react": "^7.14.5",
+    "@preact/preset-vite": "^2.1.2",
```

Comment:
Not sure but are there the React plugin for Vite?

Generally and in my opinion this new build setup should be a v2 release since this will likely break many things.
And I'm not a big fan of switching to a mix with some preact-related dependencies and a completely different build setup as for CRA there are also solutions like craco.

Also why the downgrade of react-dev-utils from v11 to v5 (which is over 3 years old)?
https://www.npmjs.com/package/react-dev-utils

Reply:
@DanielRuf This is exactly my thought as well. I need to use a major release for a change such as this. A `v2` version where I am planning to switch to`ViteJS`(or similar, newer, faster, reliable base app) and introduce multiple themes which can be used with Hyvä checkout. 

I am already working on the theme concept and probably it will also break a lot of stuff in the current version. So a major version release for such a big change + ViteJS platform makes perfect sense 👍